app/src/main/java/org/schabi/newpipe/util/StreamDialogEntry.java

```diff
@@ -1,21 +1,61 @@
 package org.schabi.newpipe.util;
 
 import android.content.Context;
+import android.widget.Toast;
 
 import androidx.fragment.app.Fragment;
 
 import org.schabi.newpipe.R;
 import org.schabi.newpipe.extractor.stream.StreamInfoItem;
 import org.schabi.newpipe.local.dialog.PlaylistAppendDialog;
+import org.schabi.newpipe.player.MainPlayer;
+import org.schabi.newpipe.player.helper.PlayerHolder;
 import org.schabi.newpipe.player.playqueue.SinglePlayQueue;
 
+import java.util.ArrayList;
 import java.util.Collections;
 
 public enum StreamDialogEntry {
     //////////////////////////////////////
     // enum values with DEFAULT actions //
     //////////////////////////////////////
 
+    /**
+     * Enqueues the stream automatically to the current PlayerType.<br>
+     * <br>
```

Comment:
br br?

Reply:
This is  just Java Doc syntax for linebreaks.

Reply:
Ah.